app/controllers/games_controller.rb

```diff
@@ -0,0 +1,49 @@
+class GamesController < ApplicationController
+
+  def index
+    @games = Game.where(user_id: current_user).order(created_at: :desc)
+  end
+
+  # all group games / all user names
+
+
+  def show
+    @game = Game.find(params[:id])
+    authorize @game
+  end
+
+   def new
+    @game = Game.new
+    authorize @game
+  end
+
+  def create
+    @game = Game.new(game_params)
+    authorize @game
+    @game.user = current_user
+    if @game.save
+      redirect_to game_path(@game)
+    else
+      render 'new'
+    end
+  # connection to group
+  end
+
+  def join
```

Comment:
im Prinzip erstellt diese Methode einfach eine `BingoCard` für `@user` und `@game`. Eventuell kann man die in einen weiteren Controller auslagern